views/js/apipCreator/test/helper/serializer.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+ define(
+    [
+        'jquery',
+        '/taoQtiItem/views/js/apipCreator/helper/serializer.js',
```

Comment:
Sorry, I didn't see it before...
Why don't you require the serializer with the amd name "taoQtiItem/apipCreator/helper/serializer" ?
Calling it this way will have the file serializer loaded each time this file is called although using the amd name with have the AMD module handled by requirejs itself (load once, cache etc.)